test/axes/baseAxisTests.ts

```diff
@@ -29,11 +29,11 @@ describe("BaseAxis", () => {
     var verticalAxis = new Plottable.Axis(scale, "right");
     verticalAxis.renderTo(svg);
 
-    var expectedWidth = verticalAxis.tickLength() + verticalAxis.margin(); // tick length and margin by default
+    var expectedWidth = verticalAxis.innerTickLength() + verticalAxis.margin(); // innerTickLength and margin by default
```

Comment:
Let's actually go ahead and remove this comment. Comments should not be necessary if the code is well-written and clear, and in this case, the comment basically says the same thing as the code preceding it almost verbatim.